client/app/billing/autoRenew/disable/disable.html

```diff
@@ -0,0 +1,19 @@
+<div data-ui-view class="container-fluid">
+    <billing-autorenew-bulk
+        data-description="billing_autorenew_disable_bulk_unavailable"
+        data-go-back="$ctrl.goBack()"
+        data-services="$ctrl.servicesList"
+        data-update-renew="$ctrl.update(services)">
+        <bulk-title>
+            <span data-translate="billing_autorenew_disable_title"></span>
+        </bulk-title>
+        <bulk-info-message>
+            <p>
+                <span class="d-block" data-translate="billing_autorenew_disable_description"></span>
+                <span class="d-block" data-translate="billing_autorenew_disable_warning"></span>
+            </p>
+            <p class="d-block" data-translate="billing_autorenew_disable_renew"></p>
```

Comment:
`<p>` element is already a block element.

```suggestion
            <p data-translate="billing_autorenew_disable_renew"></p>
```